plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/NativeClientDescriptor.java

```diff
@@ -51,14 +55,28 @@ public String getLabel() {
         return label;
     }
 
+    //Clients can't work from AppData folder, if app is Windows store app we don't want to create any native clients there
+    //see #15361
```

Comment:
This comment is pointless. It describes the result, not the reason.

Please consider attaching an MSDN page describing these limitations.

Additionally, please format this comment or, better, convert it to javadoc.